### DISCUSSION TITLE
Google Summer of Code 2022

### DISCUSSION THREAD

**Discussion by astridx** [maplibre/maplibre-gl-js#616] [Ideas]
I would like to know your opinion on[ Google Summer of Code](https://summerofcode.withgoogle.com/archive/) and whether you can imagine MapLibre [applying ](https://google.github.io/gsocguides/mentor/org-application#whats-on-the-application)there in February next year?

If you like GSoC, we should start looking/organize  ideas and mentors now.

I can think of the following for possible projects/ideas:

- Accessibility, for example make markers to buttons  (https://github.com/maplibre/maplibre-gl-js/issues/359#issuecomment-948728799)
- Performance with high pitch values  (https://github.com/maplibre/maplibre-gl-js/pull/574#issuecomment-953742943)
- Work on Browser Tests/Snapshot Testing 
- Dokumentation. Find a way to integrate our documentation here and make maintenance easier.

**Comment by wipfli** [comment]
https://github.com/maplibre/maplibre-gl-js/issues/188 could also be a nice feature to have.

**Reply by wipfli** [reply]
https://roadmap.maplibre.org/tabs/16-under-consideration
